src/components/ChatLog.jsx

```diff
@@ -2,24 +2,25 @@ import ChatEntry from './ChatEntry.jsx';
 import PropTypes from 'prop-types';
 import './ChatLog.css';
 
-const ChatLog = ({ entries, onToggleLike }) => {
+const ChatLog = ({ entries, onToggleLike = () => {} }) => {
```

Comment:
👍 Providing a default value is another useful strategy for dealing non-required props. Though in this case, consider hvaing the function still recieve the same expected params as a "real" function to help a reader see what data the function will have to work with (the id).